### PR TITLE
[pyqt-ru] New translation

### PR DESCRIPTION
**Pull request by VadimToptunov**
- [ ] I solemnly swear that this is all original content of which I am the original author
- [ ] Pull request title is prepended with `[language/lang-code]`
- [ ] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: ru-ru/[pyqt-ru] New translation [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt.py
```
Comment (vendethiel) [inline ru-ru/[pyqt-ru] New translation]: can you please use this filename for the file?

Comment (VadimToptunov) [inline ru-ru/[pyqt-ru] New translation]: Which one exactly? How exactly should I name the file? It's for the future, as I'd like to translate a couple more tutorials.

Comment (vendethiel) [inline ru-ru/[pyqt-ru] New translation]: `learnpyqt-ru.py`.

Comment (VadimToptunov) [inline ru-ru/[pyqt-ru] New translation]: Фh! Thanks! :)

Comment (vendethiel) [inline ru-ru/[pyqt-ru] New translation]: Just `pyqt-ru.html.markdown`, no `[pyqt-ru]` prefix, as you can see from the other files in that directory.

Comment (VadimToptunov) [inline ru-ru/[pyqt-ru] New translation]: ok! Thanks!

Comment (VadimToptunov) [inline ru-ru/[pyqt-ru] New translation]: Done!

File: ru-ru/pyqt-ru.html.markdown [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt-ru.py
+contributors:
+    - ["Vadim Toptunov", "https://github.com/VadimToptunov"]
```
Comment (vendethiel) [inline ru-ru/pyqt-ru.html.markdown]: can you please restore the contributors, and instead add yourself as a translator (`translators:`), like it's done https://raw.githubusercontent.com/adambard/learnxinyminutes-docs/master/ru-ru/bash-ru.html.markdown in the other files?

**Comment by VadimToptunov** [comment]
Done!

**Comment by vendethiel** [comment]
Thanks

**Pull request merged**